gdk4-wayland/src/lib.rs

```diff
@@ -8,12 +8,20 @@
 //! GDK is an intermediate layer that isolates GTK from the details of the windowing system.
 //! GDK Wayland contains functions specific to the Wayland backend.
 
+#![cfg_attr(feature = "dox", feature(doc_cfg))]
+
 pub use ffi;
 pub use gdk;
 pub use gio;
 pub use glib;
 pub use wayland_client;
 
+#[cfg(any(feature = "v4_4", feature = "dox"))]
+pub use khronos_egl;
```

Comment:
```suggestion
#[cfg(feature = "v4_4"))]
pub use khronos_egl;
```
I meant something like this

Reply:
But it'll fail with the API that use it.